Gemfile

```diff
@@ -13,7 +13,8 @@ gem "decidim-templates", path: "."
 
 gem "bootsnap", "~> 1.4"
 
-gem "puma", ">= 5.5.1"
+gem "puma", ">= 5.6.2"
+gem "rails", "~> 6.0", ">= 6.0.4.6"
```

Comment:
Any reason why doing this in the `Gemfile` and not in `decidim-core/decidim-core.gemspec`?

Reply:
No reason at all :) 

I have pushed a new version.

Reply:
Awesome, I prefer it that way, so we don't define the `rails` version in two places. Thanks!